optimum/intel/neural_compressor/quantization.py

```diff
@@ -320,7 +324,23 @@ def from_pretrained(
 
                     raise EnvironmentError(msg)
 
-            state_dict = torch.load(state_dict_path)
+            try:
```

Comment:
I think we can add a flag to specify whether or not to load a ScriptModule instead ?

Reply:
We added a config to specify the way to load.